packages/dev/core/src/Audio/v2/abstractAudioNodeParent.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable babylonjs/available */
+/* eslint-disable jsdoc/require-jsdoc */
+
+import type { AbstractAudioNode } from "./abstractAudioNode";
+import type { Nullable } from "../../types";
+import type { IDisposable } from "../../scene";
+
+export class AbstractAudioNodeParent implements IDisposable {
+    private _children: Nullable<Set<AbstractAudioNode>>;
```

Comment:
I would suggest making this readonly and initializing with a new set so that the rest of code is simpler (i.e., you don't need to check if it exists).

```suggestion
    private readonly _children = new Set<AbstractAudioNode>();
```